api/src/ApplicationError.ts

```diff
@@ -0,0 +1,41 @@
+import { ErrorCode, ERRORS, ErrorTypes } from "./errors";
+
+type ApplicationErrorOptions = {
+  isOperational: boolean;
+  exposeToClient: boolean;
+};
+
+const defaultOptions: ApplicationErrorOptions = {
+  isOperational: true,
```

Comment:
operational errors (as opposed to programmer error) are inevitable. SES might be down or a database is unavailable. You would use this to flag to support engineers "not code problem"